helm/app-operator-chart/templates/configmap.yaml

```diff
@@ -21,4 +21,4 @@ data:
       kubernetes:
         incluster: true
         watch:
-          namespace: '{{ .Values.namespace }}'
```

Comment:
I feel like this templated value could still be kept, and instead we change it in the values.yaml to be `""` ?

https://github.com/giantswarm/app-operator/blob/master/helm/app-operator-chart/values.yaml

Or otherwise we should remove it from values.yaml as well, so it's not some old code.

Reply:
But that would impact other part as well, just like `deployment.yaml` for example. That should have `giantswarm` namespace.

Reply:
I don't like that as well. We should keep that. Isn't that used by the app controller?
It should be ignored in the appcatalog controller.

Reply:
App controller also needs to watch multiple namespaces, as we want app CRs to not be in giantswarm namespace

Reply:
But I do agree that this should stay templated, just need to be careful that the deployment namespace and the watch namespace are not the same but two different template variables, having them as the same like before was wrong

Reply:
And then we need condition als if namespace is set or not

Reply:
I just pushed the changed waht @puja108 suggest me.

Reply:
Why do we need an `if` below? Empty string is fine here. Also I think it brakes formatting because there is a single space before `namespace:`.